doc/getting-started/getting-started.rst

```diff
@@ -132,31 +135,20 @@ To set up the ACRN build environment on the development computer:
       cd ~/acrn-work
       git clone https://github.com/projectacrn/acrn-hypervisor.git
       cd acrn-hypervisor
-      git checkout v3.1
+      git checkout master
 
       cd ..
       git clone https://github.com/projectacrn/acrn-kernel.git
       cd acrn-kernel
-      git checkout acrn-v3.1
+      git checkout master
 
-#. Install Python package dependencies:
 
-   .. code-block:: bash
-
-      cd ~/acrn-work/acrn-hypervisor
-      sudo python3 -m pip install -r misc/config_tools/requirements.txt
+#. Configure git with your name and email address:
 
-#. Build and install the iASL compiler/disassembler used for advanced power management,
-   device discovery, and configuration (ACPI) within the host OS:
+   .. code-block:: none
 
-   .. code-block:: bash
-
-      cd ~/acrn-work
-      wget https://acpica.org/sites/acpica/files/acpica-unix-20210105.tar.gz
-      tar zxvf acpica-unix-20210105.tar.gz
-      cd acpica-unix-20210105
-      make clean && make iasl
-      sudo cp ./generate/unix/bin/iasl /usr/sbin
+      $ git config --global user.name "David Developer"
```

Comment:
Can you remove the leading `$` sign (same for the line immediately below)?

Reply:
OK, removed.